client/lib/signup/site-type.js

```diff
@@ -75,18 +75,6 @@ export function getAllSiteTypes() {
 		},
 		{
 			id: 4,
-			slug: 'education',
-			label: i18n.translate( 'Education' ),
-			description: i18n.translate( 'Share school projects and class info.' ),
-			theme: 'pub/twentyfifteen',
-			designType: 'blog',
-			siteTitleLabel: i18n.translate( 'What is the name of your site?' ),
-			siteTitlePlaceholder: i18n.translate( 'E.g., My class' ),
-			siteTopicHeader: i18n.translate( 'What is your website about?' ),
-			siteTopicLabel: i18n.translate( 'What will your site be about?' ),
-		},
-		{
-			id: 5,
```

Comment:
I think site segments have dedicated ids, so we should keep `id: 5` for `online-store`. 

Modifying ids might cause problems as we store the segment id in the backend using `add_option` on site creation.

We also search this collection in various places [using an id value](At https://github.com/Automattic/wp-calypso/blob/ff8c882/client/signup/steps/site-type/index.jsx#L71).

This is not obvious, sorry. 🙁 Maybe we could add a comment to this effect?

Reply:
The good news is that we can remove the `education` array item in [popular-topics.jsx](https://github.com/Automattic/wp-calypso/blob/ff8c882/client/components/site-verticals-suggestion-search/popular-topics.jsx#L50), as it's only displayed when the `siteType === 'education'` 👍

Reply:
Thanks for this @ramonjd, and apologies for the late response!

I did hesitate to tweak the `id`'s for this reason! Should have followed my instincts!
I'll pop a comment in there to avoid future changes to these id's :)

Reply:
All done in https://github.com/Automattic/wp-calypso/pull/32476/commits/1f98d28e22b5b2498a908654853a1a4cb88d988b and https://github.com/Automattic/wp-calypso/pull/32476/commits/f4ae26d04576c905822cc869e8807e8a577855ec, I did a search for `getSiteTypePropertyValue` to check we weren't directly trying to get the education segment by id - looks all clear :)